packages/cli-repl/src/logger.ts

```diff
@@ -159,19 +159,20 @@ export default function logger(bus: any, logDir: string): void {
   bus.on('mongosh:api-call', function(args: ApiEvent) {
     log.info('mongosh:api-call', redactInfo(args));
 
+    // Drop analytics calls in api-call event for the time being.
     // analytics properties to include if they are present in an api-call
-    const properties: ApiEvent = {};
-    properties.arguments = {};
-    if (args.method) properties.method = args.method;
-    if (args.class) properties.class = args.class;
-    if (args.arguments) properties.arguments = properties.arguments;
-
-    if (telemetry) {
-      analytics.track({
-        userId,
-        event: 'Api Call',
-        properties: redactInfo(properties)
-      });
-    }
+    // const properties: ApiEvent = {};
```

Comment:
I think we don't need to comment, we won't resume sending events as it is done here anyway.